core/commands/files.go

```diff
@@ -70,16 +74,17 @@ operations.
 		cmds.BoolOption(filesFlushOptionName, "f", "Flush target and ancestors after write.").WithDefault(true),
 	},
 	Subcommands: map[string]*cmds.Command{
-		"read":  filesReadCmd,
-		"write": filesWriteCmd,
-		"mv":    filesMvCmd,
-		"cp":    filesCpCmd,
-		"ls":    filesLsCmd,
-		"mkdir": filesMkdirCmd,
-		"stat":  filesStatCmd,
-		"rm":    filesRmCmd,
-		"flush": filesFlushCmd,
-		"chcid": filesChcidCmd,
+		"read":         filesReadCmd,
+		"write":        filesWriteCmd,
+		"mv":           filesMvCmd,
+		"cp":           filesCpCmd,
+		"ls":           filesLsCmd,
+		"mkdir":        filesMkdirCmd,
+		"stat":         filesStatCmd,
+		"rm":           filesRmCmd,
+		"flush":        filesFlushCmd,
+		"chcid":        filesChcidCmd,
+		"replace-root": filesReplaceRoot,
```

Comment:
:thought_balloon:  
since we already use unix terms like `mkdir` `mv` `cp` `rm`, perhaps this one should be named `chroot` ?

Reply:
> since we already use unix terms like mkdir mv cp rm, perhaps this one should be named chroot ?

The current name came from an informal Slack thread, but I'm fine with whatever is decided. My 2 cents on `chroot` is that it might imply some temporary change related to the running daemon/process when in fact we're doing a long-term modification of the single filesystem available (since we don't have any jail/sandbox abstractions).

> another idea: perhaps we could add a second (optional) argument to already existing chcid, which would allow in-place swapping of CIDs for any path (not just root)

This sounds like it is beyond the scope of this issue which is fixing a corrupted filesystem preventing daemon startup (corruption in sub-paths wouldn't have the same effect). The idea is valid on itself though, maybe we should discuss it in another issue as a feature request.